DeveloperDeployment.md

```diff
@@ -1,4 +1,4 @@
 # Developer Deployment Guide
 As we add new features or fix bugs, we will need to deploy the changes to the RunbookEntrypointManifest and any files referenced by that manifest. The issue is that the RunbookEntrypoint script needs to retrieve this manifest and referenced files from the corresponding developer branch. In order to do this, we have added a variable to the RunbookEntrypoint script. This variable should be set to the name of the branch that contains the manifest and referenced files. The RunbookEntrypoint script will retrieve the manifest and referenced files from this branch.
 
-To deploy a script from a cutom branch, simply go to the branch and deploy from the AzureSqlBulkFailoverSetup.md instructions in that branch. This will redirect you to a DeployTemplate.html file that will deploy the arm template from your branch to your azure subscription. Ensure you reenter your custom branch name in the "BranchName" variable in the arm template deployment page and also in the "BranchName" variable in the RunbookEntrypoint script when running the runbook.
\ No newline at end of file
+To deploy a script from a custom branch, simply go to the branch and deploy from the AzureSqlBulkFailoverSetup.md instructions in that branch. This will redirect you to a DeployTemplate.html file that will deploy the arm template from your branch to your azure subscription. Ensure you reenter your custom branch name in the "BranchName" variable in the arm template deployment page and also in the "BranchName" variable in the RunbookEntrypoint script when running the runbook.
```

Comment:
Add: "Also note that the branch must have public access so that the runbook can access it."

Reply:
Done